tests/testCatalogDBObject.py

```diff
@@ -502,6 +519,243 @@ def testQueryColumnsDefaults(self):
                 self.assertEqual(line[2], controlArr[ix][2])
 
 
+    # The tests below all replicate tests above, except with CatalogDBObjects whose
+    # connection was passed directly in from the constructor, in order to make sure
+    # that passing a connection in works.
+
+    def testNonsenseCircularConstraints_passConnection(self):
+        """
+        Test that a query performed on a circle bound gets all of the objects (and only all
+        of the objects) within that circle.
+
+        Pass connection directly in to the constructor.
+        """
+
+        myNonsense_base = CatalogDBObject.from_objid('Nonsense')
+        myNonsense = myNonsenseDB_noConnection(connection=myNonsense_base.connection)
```

Comment:
Looks like this is answered and tested in the related pull request for sims_catalogs_measures. Am I interpreting that right? If not, let me know.